examples/mnist/quantized.cpp

```diff
@@ -1,124 +0,0 @@
-/*
```

Comment:
@edgarriba Thank you for the explanation. It was my mistake to delete it. Sorry!

I noticed in kernel tiny_quantized_fully_connected_kernel and tiny_quantized_fully_connected_back_kernel, float_tensor_to_quantized is called frequently.
I think calling conversions should be kept minimum to sustain performance, but to do so, I suppose changing data structure is needed to hold quantized data.